app/controllers/hyrax/admin/admin_sets_controller.rb

```diff
@@ -22,10 +22,9 @@ class Admin::AdminSetsController < ApplicationController
     self.admin_set_create_service = AdminSetCreateService
 
     def show
-      add_breadcrumb t(:'hyrax.controls.home'), root_path
-      add_breadcrumb t(:'hyrax.dashboard.breadcrumbs.admin'), hyrax.dashboard_path
-      add_breadcrumb t(:'hyrax.admin.sidebar.admin_sets'), hyrax.admin_admin_sets_path
-      add_breadcrumb t(:'hyrax.admin.admin_sets.show.breadcrumb'), request.path
+      add_breadcrumb t(:'hyrax.dashboard.breadcrumbs.dashboard'), hyrax.dashboard_path
```

Comment:
Is there a reason not to use `hyrax.toolbar.dashboard.menu` here?

Reply:
Thanks @jcoyne I'll check that option out.

Reply:
Yes, that one that @elrayle suggests is an even better option.

Reply:
Resovled